File: greenDao/src/main/java/github/com/greendao/sql/MyGreenDaoGenerator.java
package github.com.greendao.sql;

import de.greenrobot.daogenerator.DaoGenerator;
import de.greenrobot.daogenerator.Entity;
import de.greenrobot.daogenerator.Schema;

/**
 * by y on 2016/10/8
 */

public class MyGreenDaoGenerator {

    public static void main(String[] args) throws Exception {
        //版本号,包名
        Schema schema = new Schema(1, "github.com.greendao.sql");
        Entity user = schema.addEntity("User");
        user.addIdProperty().primaryKey();
        user.addStringProperty("userName").notNull();
        user.addStringProperty("userSex").notNull();
        new DaoGenerator().generateAll(schema, "./app/src/main/java");
    }

}

File: GreenDao/2x/src/main/java/greendao/develop/android/two/MyGreenDaoGenerator.kt
package greendao.develop.android.two

import de.greenrobot.daogenerator.DaoGenerator
import de.greenrobot.daogenerator.Schema

object MyGreenDaoGenerator {

    @Throws(Exception::class)
    @JvmStatic
    fun main(args: Array<String>) {
        //版本号,包名
        val schema = Schema(1, "greendao.develop.android.two")
        val user = schema.addEntity("User2x")
        user.addIdProperty().primaryKey()
        user.addStringProperty("userName2x").notNull()
        user.addStringProperty("userSex2x").notNull()
        DaoGenerator().generateAll(schema, "./2x/src/main/java")
    }

}
